### ISSUE TITLE
Translation of yet another new label

### ISSUE THREAD

**Issue by gildas-lormeau** [gildas-lormeau/SingleFile#177]
Hi @KrasnayaPloshchad, @yfdyh000, @Shitennouji, @xesarni, @rstp14, @frostblazergit

I added a new option to save pages to the clipboard instead of downloading them. This fixes the issue #163 and introduces one new entry in the `messages.json` file:

`optionSaveToClipboard: "save to clipboard"`

As usual, your help in translating this label would be greatly appreciated!

Status:

- [x] German
- [x] English
- [x] French
- [x] Japanese
- [x] Chinese
- [ ] Russian
- [x] Polish

**Comment by gildas-lormeau** [comment]
@womotroll I did the German translation, cf. https://github.com/gildas-lormeau/SingleFile/blob/master/_locales/de/messages.json#L355. Please let me know if that's fine for you.

**Comment by frostblazergit** [comment]
Chinese:
`optionSaveToClipboard: "保存到剪切板"`

Improve translation:

```
"buttonOKBadge": "完成"
"optionAutoSaveDelay": "页面加载后自动保存延时（秒）"
"optionAutoSaveRepeatDelay": "定期自动保存周期（秒）"
```

I found that the "description" attribute seems to only describe "options*", which may be better reserved for English.  If necessary, please let me know and I will translate it.

**Comment by frostblazergit** [comment]
Add a brief description of the current language to each language translator in the "`Contributors`" node of the github and browser extensions store to make it better searchable (SEO: Search Engine Optimization).

e.g.

### Contributors
Chinese translation done by ...

Chinese function summary:
SingleFile 网页保存（Save Page），可以更好的替代已经停止维护的“Mozilla Archive Format”Firefox扩展。离线保存当前网页、多个标签页、当前选中的内容，更多高级功能例如：自动保存、定期保存，等待你的探索！`English(delete this):  The SingleFile Web Page Save (Save Page) is a better alternative to the "Mozilla Archive Format" Firefox extension that has been discontinued. Save the current web page, multiple tabs, currently selected content offline, and more advanced features such as: auto save, save regularly, wait for your exploration!`


Japanese translation done by ...

**Comment by Shitennouji** [comment]
Japanese：
optionSaveToClipboard: "クリップボードに保存する"

**Comment by xesarni** [comment]
Polish:
optionSaveToClipboard: "zapisz do schowka"

**Comment by gildas-lormeau** [comment]
Thank you all for your swift responses!

@frostblazergit Thank you also for your proposal. I almost agree with your text. The only problem for me is that it is very subjective to say that SingleFile is a "better" alternative to MAF. Could you provide the same text without the word "better"? I think I will create a localized extension page for Chinese with this description at the top, and the instructions in English below.

**Comment by frostblazergit** [comment]
@gildas-lormeau 
SingleFile 网页保存（Save Page）扩展，离线保存当前网页、多个标签页、当前选中的内容，更多高级功能例如：自动保存、定期保存，等待你的探索！`SingleFile Web Page Save (Save Page) extension, offline save the current web page, multiple tabs, currently selected content, more advanced features such as: automatic save, regular save, waiting for your exploration!`

or

SingleFile 网页保存（Save Page）扩展，类似“Mozilla Archive Format”Firefox扩展。离线保存当前网页、多个标签页、当前选中的内容，更多高级功能例如：自动保存、定期保存，等待你的探索！`The SingleFile Save Page extension, similar to the "Mozilla Archive Format" Firefox extension. Save the current web page, multiple tabs, currently selected content offline, and more advanced features such as: auto save, save regularly, wait for your exploration!`

Perhaps it is better to remove "MAF" so that users don't have questions: What does the "MAF" extension do? Can it still be used now? It is up to you to decide. : )

**Comment by rstp14** [comment]
Russian:
optionSaveToClipboard: "сохранить в буфер обмена"

**Comment by gildas-lormeau** [comment]
@frostblazergit I updated the description of the extension. I decided to integrate your first proposition. Thanks!

@rstp14 Thank you also for your swift response!

**Comment by frostblazergit** [comment]
> Chinese:
> `optionSaveToClipboard: "保存到剪切板"`
> 
> Improve translation:
> 
> ```
> "buttonOKBadge": "完成"
> "optionAutoSaveDelay": "页面加载后自动保存延时（秒）"
> "optionAutoSaveRepeatDelay": "定期自动保存周期（秒）"
> ```

@gildas-lormeau 
`"optionAutoSaveRepeat"` was incorrectly replaced with `"optionAutoSaveRepeatDelay"`
![image](https://user-images.githubusercontent.com/48089979/54890743-aab55a80-4ee5-11e9-8cfc-b72b07ab05b3.png)


The correct one should be:
```
"optionAutoSaveRepeat": "定期自动保存"
"optionAutoSaveRepeatDelay": "定期自动保存周期（秒）"
```

I hope this bug can be fixed in the next update.

**Comment by gildas-lormeau** [comment]
Thanks @frostblazergit! It's fixed in the new version 1.10.4 I just pusblished
